go.mod

```diff
@@ -128,6 +128,8 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require github.com/stretchr/testify v1.8.4
```

Comment:
```suggestion
```

We don't use testify, we utilize the builting go test generation provided by https://github.com/cweill/gotests which relies on `reflect.DeepEqual` usually. 
See https://github.com/rancher/rke2/blob/master/pkg/rke2/serviceaccount_test.go#L127 and https://github.com/rancher/rke2/blob/master/pkg/rke2/rke2_linux_test.go#L138-L142